src/components/InngestStepTools.ts

```diff
@@ -369,16 +369,18 @@ export const createStepTools = <
         /**
          * The date to wait until before continuing.
          */
-        time: Date
+        time: Date | string
       ) => void
     >((time) => {
+      const date: Date = typeof time === "string" ? new Date(time) : time;
```

Comment:
I think `new Date("some-bad-string")` will not throw and instead return a crappy "Invalid Date" object.

You can check for this with `date.toString() === "Invalid Date"` and I guess we can throw here?